.github/workflows/unit-tests.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - main
+      - dev
```

Comment:
Would it be helpful to run these on any branch during development?

Reply:
This runs on pull requests already, so I interpret this as another safety run essentially.

```
  pull_request:
    types: [opened, synchronize, reopened]
```

Reply:
Got it, thanks!